packages/app/prisma-utils/src/prismaClientFactory.spec.ts

```diff
@@ -17,7 +17,7 @@ describe('prismaClientFactory', () => {
       })
 
       expect(prisma).toBeDefined()
-      expect(await prisma.item1.findMany()).toHaveLength(0)
+      expect(await prisma.item1.findMany()).toBeDefined()
```

Comment:
Just fixing flaky test, prisma transaction tests are affecting this one, as we don't care about results and we only want to check that the client works so we can avoid a DB clean-up here